examples/full_example/main.tf

```diff
@@ -82,6 +84,10 @@ module "full" {
   user_arns                                    = local.user_arns
   viewer_base_policy_arn                       = "arn:aws:iam::aws:policy/ReadOnlyAccess"
   viewer_custom_policy_json                    = ""
+  enable_cloudwatch_reporting_role             = true
+  cloudwatch_reporting_base_policy_arn         = "arn:aws:iam::aws:policy/CloudWatchReadOnlyAccess"
```

Comment:
This role would allow you to get logs if not explicitly denied by another statement in a merged policy. (assuming uptime stats are coming from metrics) perhaps a custom policy granting access only to what's needed would be better (given this is running in prod accounts).

GetMetricData or GetMetricStream?

https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazoncloudwatch.html